src/main/scala/com/fulcrumgenomics/sv/tools/AggregateSvPileup.scala

```diff
@@ -0,0 +1,319 @@
+package com.fulcrumgenomics.sv.tools
+
+import com.fulcrumgenomics.FgBioDef.{FgBioEnum, FilePath}
+import com.fulcrumgenomics.sopt.{arg, clp}
+import com.fulcrumgenomics.sv.BreakpointPileup
+import com.fulcrumgenomics.sv.cmdline.{ClpGroups, SvTool}
+import com.fulcrumgenomics.sv.tools.AggregatedBreakpointPileup.addId
+import com.fulcrumgenomics.sv.tools.BreakpointCategory.BreakpointCategory
+import com.fulcrumgenomics.util.{Io, Metric}
+
+import scala.annotation.tailrec
+import scala.collection.mutable
+
+
+/** Type aliases */
+trait AggregateSvPileupDef {
+  type PileupGroup = Seq[BreakpointPileup]
+}
+
+
+@clp(group=ClpGroups.All, description=
+  """
+    |Merges nearby pileups of reads supporting putative structural variants.
+    |
+    |Takes as input the file of pileups produced by `SvPileup`. That file contains a list of breakpoints, each
+    |consisting of a chromosome, position and strand for each side of the breakpoint, as well as quantified read support
+    |for the breakpoint.
+    |
+    |This tool merges pairs of breakpoints that have their left and right ends on the same chromosomes, the same left
+    |and right strands, and their left and right positions both within a length threshold. The merging behavior is
+    |transitive. For example, two breakpoints that are farther apart than the length threshold can be merged if there
+    |is a third breakpoint that is close to both.
+    |
+    |The output file is a tab-delimited table with one record per aggregated cluster of pileups. Aggregated
+    |pileups are reported with the minimum and maximum (inclusive) coordinates of all pileups in the cluster, a
+    |possible putative structural variant event type supported by the pileups, and the sum of read support from all
+    |pileups in the cluster.
+    |""")
+class AggregateSvPileup
+(@arg(flag='i', doc="The input text file of pileups generated by SvPileup") input: FilePath,
+ @arg(flag='o', doc="The output file") output: FilePath,
+ @arg(flag='d', doc="Distance threshold below which to cluster breakpoints") maxDist: Int = 100,
```

Comment:
I wonder if we want a pair of thresholds?  One for merging split-reads, and one for merging paired-end?  I would _think_ it would make sense to have a pretty low threshold (e.g 10-20bp) for differences in split-read positions, but a _much_ larger one (e.g. 250bp) for paired-end/read-pair?